cgnsutilities/cgnsutilities.py

```diff
@@ -584,7 +584,7 @@ def simpleCart(self, dh, hExtra, nExtra, sym, mgcycle, outFile):
         # Call the generic routine
         return simpleCart(xMin, xMax, dh, hExtra, nExtra, sym, mgcycle, outFile)
 
-    def simpleOCart(self, dh, hExtra, nExtra, sym, mgcycle, outFile, comm=None):
+    def simpleOCart(self, dh, hExtra, nExtra, sym, mgcycle, outFile, comm=None, userOptions=None):
```

Comment:
perhaps naming this `hypOptions` and renaming the variable below to `defaultHypOptions` or something would be a bit clearer?

Reply:
I update the defaults with the user options, so I don't want the final options that get passed to pyHyp to be named `defaultHypOptions`. I could change `userOptions` to `userHypOptions`?